data/metadata.json

```diff
@@ -1156,6 +1156,77 @@
     "quests/transfer-complete": {
         "sound-file": "quests/transfer-complete.webm"
     },
+    "sidetrack/bg/manual_mode": {
+        "sound-file": "sidetrack/bg/bg1.wav",
```

Comment:
Please convert wav files to webm.
Use `./tools/convert-sounds` script.

Reply:
wav files converted in latest commit.